src/mds/agg_mds/datastore/elasticsearch_dao.py

```diff
@@ -195,9 +392,9 @@ async def get_all_named_commons_metadata(name):
         return {}
 
 
-async def metadata_tags(name):
+async def metadata_tags():
```

Comment:
looks like a breaking change in the data returned by this endpoint. Will that impact HEAL?

Reply:
HEAL is not using this option